Refresh.GameServer/Endpoints/ApiV3/DataTypes/Response/ApiExtendedGameUserResponse.cs

```diff
@@ -26,6 +26,8 @@ public class ApiExtendedGameUserResponse : IApiResponse, IDataConvertableFrom<Ap
     public required bool RpcnAuthenticationAllowed { get; set; }
     public required bool PsnAuthenticationAllowed { get; set; }
     
+    public bool RedirectGriefReportsToPhotos { get; set; } 
```

Comment:
Since we declare a new option on the user here, we're gonna need a corresponding `refresh-web` change. Should be trivial enough.